src/layouts/components/Editor/components/BlockWrapper/BlockWrapper.tsx

```diff
@@ -31,6 +31,7 @@ export const BlockWrapper = ({
           backgroundColor="#055AFF"
           textColor="white"
           p="0.25rem"
+          mt="-0.75rem"
```

Comment:
why's this required? from the [review](https://www.notion.so/opengov/Accordion-design-QA-7a07a952d7514f6989e8d04f61c1fac8?pvs=4#f499625ebfa14e94aff1662b73d6e345), this seems like accordion should have a `0.75rem` gap, not that the gap between elements should be reduced by `0.75rem`